### ISSUE TITLE
[1.5.0-17b] Arcane Knowledge is unlocked by default

### ISSUE THREAD

**Issue by xenxero1** [Growlith1223/ArsMagica2#207]
Upon loading into a server, every one of my players has their arcane knowledge unlocked, and can see their magic level, mana + burnout bars, and when reading the Arcane Compendium, all knowledge is unlocked outside of spell components)

Using the Spinycraft V2 pack on Curse.

**Comment by xenxero1** [comment]
I will also add, In other older versions of AM2, all compendium knowledge was sometimes unlocked by default. However, I can't remember it ever displaying magic level and bars pre-compendium.

**Comment by EPIICTHUNDERCAT** [comment]
Version?

**Comment by xenxero1** [comment]
Current curse-forge release

**Comment by TheIcyOne** [comment]
That has a tendency to change quite rapidly, can you give a specific version number?

**Comment by xenxero1** [comment]
ArsMagica2-1.10.2-1.5.0-17b.jar

**Comment by EPIICTHUNDERCAT** [comment]
yes always specify versions, also like i said test update versions to make sure the issue still exists!

**Comment by FunnyBunny14** [comment]
I also have this problem in my single player survival world. Other than spells, all entries were unlocked upon creation of the book (bosses, etc.). It's kinda annoying considering I'm new to this mod and actually wanted to have to unlock all the entries one by one, in order to properly learn it. 
Version: ArsMagica2-1.10.2-1.5.0-18

**Comment by ImperfMage** [comment]
I have also come across this problem, Im new as well .. just all confusing to see at once
